example/install_src.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+PWD=$(pwd)
+MYDIR=${1:-$PWD}
+if [ ! -d $MYDIR ]; then
+    echo "[Error] working directory=$MYDIR doesn't exist"
+    exit -1
+fi
+
+pushd $MYDIR
+
+WORKDIR=$(pwd)  # Absolute Path
+
+
+set -e
+set -x
+
+BRANCH=${2:-master} # can be a tag like 1.0.0
+
+
+
+NODE_CORE_REPOS=("on-core" "on-tasks" "di.js" )
+NODE_OTHER_REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog" )
+NODE_REPOS=("${NODE_CORE_REPOS[@]}" "${NODE_OTHER_REPOS[@]}")
+OTHER_REPOS=( "on-wss" "on-tools" "on-imagebuilder" "RackHD")
+REPOS=( "${NODE_REPOS[@]}"  "${OTHER_REPOS[@]}" )
+#GITHUB="https://eos2git.cec.lab.emc.com/RackHD" # https://github.com/RackHD
+GITHUB="https://github.com/RackHD"
+
+
+echo "[Info] Clone RackHD repos, and checkout to  branch ${BRANCH}"
+
+
+for r in ${REPOS[@]}; do
+    rm ${r} -rf
+    git clone ${GITHUB}/${r}.git
+    pushd ${r}
+    git fetch --all --prune --tags
+    if [ -z $BRANCH ]; then
+        git checkout ${BRANCH}
+    fi
+    popd
+done
+
+for r in ${NODE_REPOS[@]}; do
+    pushd ${r}
+    npm install --production
+    popd
+done
+
+echo "[Info] Make common static directory & generate Docs"
+HTTP_STATIC_FOLDER=on-http/static/http
+TFTP_STATIC_FOLDER=on-tftp/static/tftp
+mkdir -p $HTTP_STATIC_FOLDER
+mkdir -p $TFTP_STATIC_FOLDER
+mkdir -p on-http/static/http/common
+pushd on-http
```

Comment:
I followed the steps in https://github.com/RackHD/RackHD/blob/master/packer/ansible/roles/remoterepos/tasks/main.yml

so I believe that it is